.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+Thanks for raising an issue! To help us help you, if you've found a bug please consider the following:
+
+* If you can demonstrate the bug using the interactive [rollupjs.org](http://rollupjs.org) demo, please do.
```

Comment:
Since this is commented out, should we bother with markdown? This isn't a "template" so much as notes on how to submit an issue, which is fine, but it makes the inclusion of markdown seem out of place.